.ci/on-schedule.sh

```diff
@@ -107,6 +109,46 @@ function collect_aur_timestamps() {
     http_proxy="$CI_AUR_PROXY" https_proxy="$CI_AUR_PROXY" UTIL_FETCH_AUR_TIMESTAMPS collect_aur_timestamps_output "${AUR_PACKAGES[*]}"
 }
 
+function collect_changed_libs() {
+    for repo in "${CI_ARCH_REPOS[@]}"; do
+        UTIL_PARSE_DATABASE "$repo" "${CI_ARCH_MIRROR}/$repo/os/x86_64/${repo}.db"
+    done
+    UTIL_PARSE_DATABASE "${REPO_NAME}" "${CI_CAUR_MIRROR}/${REPO_NAME}/x86_64/${REPO_NAME}.db"
```

Comment:
I thought we were saying we wouldn't do this? If you insist, I would prefer if you made this variable 100% optional (that includes setting it correctly in util.shlib)

Reply:
Added an array with links to dbs as a more flexible solution.